explorer/nodewatch/tests/test_RoutesFacade.py

```diff
@@ -428,7 +428,7 @@ def test_can_generate_height_chart_with_metadata_json(self):
 
 		# Assert: { height, finalized_height } x { 1.0.3.4, 1.0.3.5 }
 		self.assertEqual(2, len(chart_with_metadata_json))
-		self.assertEqual(4, len(chart_json['data']))
+		self.assertEqual(6, len(chart_json['data']))
 		self.assertTrue(re.match(r'\d\d:\d\d', chart_with_metadata_json['lastRefreshTime']))
 
 	def test_can_retrieve_estimated_network_height_json(self):
```

Comment:
please add tests for new filtering logic

Reply:
I miss the unit test in this section 👍🏼